tests/python/pocketoption/test_async_mocked.py

```
@@ -595,7 +595,7 @@ async def test_get_candles_live_success(self, async_client, mock_raw_pocketoptio
             )
         )
         gen = async_client.get_candles_live("EURUSD_otc", period=60, hours=0.1, max_rows=10)
-        closed, forming = await anext(gen)
+        closed, forming = await gen.__anext__()
```

Ruff (0.16.3)

[warning] Unpacked variable `forming` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)